sdk/servicebus/azure-servicebus/azure/servicebus/_transport/_pyamqp_transport.py

```diff
@@ -567,6 +571,23 @@ def create_receive_client(
         config = receiver._config   # pylint: disable=protected-access
         source = kwargs.pop("source")
         receive_mode = kwargs.pop("receive_mode")
+        link_properties = kwargs.pop("link_properties")
+
+        if receiver._session_id == NEXT_AVAILABLE_SESSION and receiver._max_wait_time: # pylint: disable=protected-access
```

Comment:
can we add a comment explaining why only for NEXT_AVAILABLE_SESSION?

something like:
- This is specifically for the scenario where: operation timeout is _only_ returned by the service if 1) the queue is empty and 2) on initial connection. If there's a single message, but the queue is empty until max_wait_time reached after, a different error is raised.

Reply:
(basically just a small summary of what you have in the PR description but as a comment here 🙂 )